### PR TITLE
Fix/2018 task list indication

### PR DESCRIPTION
**Pull request by max-nextcloud**
* Resolves: #2018
* Target version: master 

### Summary

* Use tiptaps TaskList and BulletList with their respective items.
  This fixes the indicator of which list type is active.
* Split mixed lists in markdown-it already.
* Use an icon that is more similar to the other list icons for the tasklist.

https://user-images.githubusercontent.com/97337118/154509747-7160acb2-2734-43f6-afe8-87bb1c9e055b.mp4

### CODE REVIEW

File: img/tasklist.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="24" height="24" viewBox="0 0 24 24"><path d="M4 13C2.89 13 2 13.89 2 15V19C2 20.11 2.89 21 4 21H8C9.11 21 10 20.11 10 19V15C10 13.89 9.11 13 8 13M8.2 14.5L9.26 15.55L5.27 19.5L2.74 16.95L3.81 15.9L5.28 17.39M4 3C2.89 3 2 3.89 2 5V9C2 10.11 2.89 11 4 11H8C9.11 11 10 10.11 10 9V5C10 3.89 9.11 3 8 3M4 5H8V9H4M12 5H22V7H12M12 19V17H22V19M12 11H22V13H12Z" /></svg>
```
Comment (juliushaertl) [inline img/tasklist.svg:1]: Do you have a reference where that icon is from, especially in terms of the license?

Comment (max-nextcloud) [reply img/tasklist.svg:1]: It's the icon proposed in the issue. "order-bool-ascending-variant" downloaded from https://materialdesignicons.com/.
The same exists in as OrderBoolAscendingVariant in `vue-material-design-icons`. However I did not know how to include it in the menubar.js.

Comment (juliushaertl) [reply img/tasklist.svg:1]: Ok, perfect.

**Review by juliushaertl — COMMENTED** [review]
Looks good code wise, will give it another test run as discussed after the rebase :)

**Comment by juliushaertl** [comment]
Ah, GitHub hasn't updated by tab, let me test then right away

**Pull request merged**